### PR TITLE
Fe/txn progress bar

### PR DESCRIPTION
**Pull request by bigboydiamonds**
## Summary by CodeRabbit

- **New Features**
	- Introduced an animated progress bar to visually represent transaction progress within the `_Transaction` component.
	- Added a `TestProgressBar` component for testing progress bar functionality with user-defined inputs.

36c1df4dd876775404dab76e7391bdc5d20b9b9a: [synapse-interface preview link ](https://sanguine-synapse-interface-glv21kriu-synapsecns.vercel.app)
d085348f4ee39106b951e9910797c90c0a0b1e3b: [synapse-interface preview link ](https://sanguine-synapse-interface-pmqp4olrz-synapsecns.vercel.app)
0156c692b934ea9487c80e5f0e2343df82538761: [synapse-interface preview link ](https://sanguine-synapse-interface-4v8fwcncr-synapsecns.vercel.app)
d9ebba51cdac451057556557b9f66392ff5d2741: [synapse-interface preview link ](https://sanguine-synapse-interface-ckz7ex2w9-synapsecns.vercel.app)
6a584c255e0c331263a1bd6586886d3868a39af9: [synapse-interface preview link ](https://sanguine-synapse-interface-jns3iv8s3-synapsecns.vercel.app)
3fefaf88ddf2e797b80e75a23734f22cce9b0bf3: [synapse-interface preview link ](https://sanguine-synapse-interface-q7gcohz79-synapsecns.vercel.app)
591d5c95791bb6db2f7ba02542cb1168247258f4: [synapse-interface preview link ](https://sanguine-synapse-interface-ychevf0zj-synapsecns.vercel.app)
3ea39b141cd7513410bf0054bcd82e21b0dfd513: [synapse-interface preview link ](https://sanguine-synapse-interface-2i0x6iifm-synapsecns.vercel.app)
290f9d15ad0e623e4c57ffd85cdf5d11b7290d00: [synapse-interface preview link ](https://sanguine-synapse-interface-bfy1tqm7e-synapsecns.vercel.app)
a01a7204d4e74b70888242b201df9408a12874dd: [synapse-interface preview link ](https://sanguine-synapse-interface-7k9b0i8os-synapsecns.vercel.app)
cdd932d08da87d3145145193fe805ee6b7aff002: [synapse-interface preview link ](https://sanguine-synapse-interface-7gnjp0ewo-synapsecns.vercel.app)
bb6402233d2b54a8f928b3ec1fab0baa2fa96e12: [synapse-interface preview link ](https://sanguine-synapse-interface-q4inwkx3x-synapsecns.vercel.app)
fdb32a3af2b02a143cead22eb17c1e26b58db9ad: [synapse-interface preview link ](https://sanguine-synapse-interface-8j7j5jnzs-synapsecns.vercel.app)
246f4309a19a783490cdcb81e66f3ce7b09771d5: [synapse-interface preview link ](https://sanguine-synapse-interface-b7l15bv3x-synapsecns.vercel.app)
c7b1f140212e244da173a082f478450435b2af07: [synapse-interface preview link ](https://sanguine-synapse-interface-h7gmyglzq-synapsecns.vercel.app)
d6ee6bff9b4489df28606297a48ddcba55fa5671: [synapse-interface preview link ](https://sanguine-synapse-interface-8cuvogs93-synapsecns.vercel.app)
e1ae4ddfe2cbfbc706aae7ba7d54cf003a959d25: [synapse-interface preview link ](https://sanguine-synapse-interface-91qob5abq-synapsecns.vercel.app)
6dbe70afbffcbdd0737e22a2919768c3291805f6: [synapse-interface preview link ](https://sanguine-synapse-interface-inugas2ep-synapsecns.vercel.app)
b701c9a782a9eef16955c3ca74776108cb1aedd4: [synapse-interface preview link ](https://sanguine-synapse-interface-cimw5vscx-synapsecns.vercel.app)
aee3f00c60f196ab3ce4a86f8cdd4f157c090eef: [synapse-interface preview link ](https://sanguine-synapse-interface-97u0uu8bp-synapsecns.vercel.app)
d88d2997c79ac1bf9d95bb32e9cf302783420e13: [synapse-interface preview link ](https://sanguine-synapse-interface-l6mtezluy-synapsecns.vercel.app)

### CODE REVIEW

File: packages/synapse-interface/assets/icons/susd.svg
Comment (abtestingalpha) [inline packages/synapse-interface/assets/icons/susd.svg:1]: Build failing bc this file got deleted

Comment (bigboydiamonds) [reply packages/synapse-interface/assets/icons/susd.svg:1]: Don't know how that got deleted! Added back

**Comment by bigboydiamonds** [comment]
merged into https://github.com/synapsecns/sanguine/pull/2002

**Comment by bigboydiamonds** [comment]
Closing - merged in https://github.com/synapsecns/sanguine/pull/2002

**Pull request closed without merging**